.evergreen/run-tests.sh

```diff
@@ -170,11 +172,27 @@ if [ -n "$TEST_ENCRYPTION" ]; then
         # Remove after BUILD-13574.
         python -m pip install certifi
     fi
+    # The mock KMS server requires Python >=3.5 with boto3.
+    IS_PRE_35=$(python -c "import sys; sys.stdout.write('1' if sys.version_info < (3, 5) else '0')")
+    if [ $IS_PRE_35 = "1" ]; then
+        deactivate
+        createvirtualenv $PYTHON3_BINARY venv-kms
+        python -m pip install boto3
+    fi
     pushd ${DRIVERS_TOOLS}/.evergreen/csfle
     python -u lib/kms_http_server.py --ca_file ../x509gen/ca.pem --cert_file ../x509gen/expired.pem --port 8000 &
     python -u lib/kms_http_server.py --ca_file ../x509gen/ca.pem --cert_file ../x509gen/wrong-host.pem --port 8001 &
     trap 'kill $(jobs -p)' EXIT HUP
     popd
+    # Restore the test virtualenv.
+    if [ $IS_PRE_35 = "1" ]; then
```

Comment:
Can we add an `activatevritualenv` function to `utils.sh` and just call that from both `createvirtualenv` and also here? It'll be much cleaner.

Reply:
Done.